apps/prairielearn/src/middlewares/authn.ts

```diff
@@ -46,11 +47,13 @@ export default asyncHandler(async (req, res, next) => {
 
     const uuid = data.uuid;
 
-    let authnParams = {
+    const authnParams = {
```

Comment:
Maybe add a type annotation here?

```suggestion
    const authnParams: LoadUserAuth = {
```